.github/workflows/CompatHelper.yml

```diff
@@ -0,0 +1,26 @@
+name: CompatHelper
+
+on:
+  schedule:
+    - cron: '00 * * * *'
+  issues:
+    types: [opened, reopened]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        julia-version: [1.5.0]
+        julia-arch: [x86]
+        os: [ubuntu-latest]
+    steps:
+      - uses: julia-actions/setup-julia@latest
```

Comment:
As github action now ships a default Julia, there is no need to install it again and we could remove this step.

See also https://github.com/JuliaRegistries/CompatHelper.jl#11-create-the-workflow-file-required

Reply:
Ah, I should have just used the standard one instead of copying from some other repo!  Thanks!
Note to self: I also created and deployed the ssh key as described in the instructions.